### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by wborn**
This makes reading the docs a bit easier. :slightly_smiling_face:

### CODE REVIEW

File: developers/bindings/index.md [hunk 1]
```diff
@@ -966,7 +966,7 @@ mDNS discovery is implemented in the framework as `MDNSDiscoveryService`.
 To facilitate the development, binding developers only need to implement a `MDNSDiscoveryParticipant`.
 Here the developer only needs to implement four simple methods:
 
-- `getServiceType` - Defines the [mDNS service type](https://www.dns-sd.org/ServiceTypes.html).
+- `getServiceType` - Defines the [mDNS service type](http://www.dns-sd.org/ServiceTypes.html).
```
Comment (Confectrician) [inline developers/bindings/index.md:969]: Sad to have to revert https urls in 2022, but it seems correct unfortunately. 🤷‍♂️

**Review by Confectrician — APPROVED** [review]
Thanks for tha many findings. :)

**Comment by wborn** [comment]
I got a little bit of help of a broken URL finder. 😉

**Comment by Confectrician** [comment]
As external tool?
I know that we have tried to implemeent this as a script/action for internal ones but we never did it on a regular basis and i think also that it still was for the old website system.

**Comment by wborn** [comment]
I didn't spend a lot of time on it and tried a couple of websites that do this and this one worked pretty well: https://www.deadlinkchecker.com/website-dead-link-checker.asp (it has a 2000 link limit like most of these online tools have).

It might also be an option to see if the [LinkChecker](https://wummel.github.io/linkchecker/index.html) tool used in #710 works well with the current website. If it doesn't, that site also lists some [other similar tools](https://wummel.github.io/linkchecker/other.html). :slightly_smiling_face:  

I also found a GitHub Action that can be used to check broken links with lychee: https://github.com/lycheeverse/lychee-action

**Pull request merged**